README.md

```diff
@@ -1,33 +1,37 @@
 # AerialWaste dataset
 
-AerialWaste is a dataset for the discovery of illegal landfill. Illegal landfills from aerial images present a visual heterogeneity of the scenes in which waste dumps appear and present a diverse nature of the objects that compose a waste deposit. When observed from above waste dumps appear as complex arrangements of objects of different shapes, sizes, and orientation.
+AerialWaste is a dataset for the discovery of illegal landfills. Illegal landfills from aerial images present a visual heterogeneity of the scenes in which waste dumps appear and present a diverse nature of the objects that compose a waste deposit. When observed from above, waste dumps appear as complex arrangements of objects of different shapes, sizes, and orientation.
 
 Visit our site for more details: https://aerialwaste.org/
 
 ## Repository content
 
-This repository contains a serie of utilities scripts to handle the dataset:
-  - Link to dataset: Download images from Zenodo[https://zenodo.org/record/7034382], put the training.json and testing.json containing the images medatadata in the root of this repository and put all images into an images folder.
+This repository contains a series of utility scripts to handle the dataset:
+  - Link to dataset: Download images from Zenodo[https://zenodo.org/record/7034381], put the `training.json` and `testing.json` containing the images medatadata in the root of this repository and put all images into an `images` folder.
   - Statistics: to plot statistics on the dataset.
-  - Visualizer: to visualize the images with its correspondent classes and segmentation masks. Download and unzip image folder, and install ODIN visualizer tool https://github.com/rnt-pmi/odin). 
-  -  DataLoader: to conver the JSON to tensors containing the image classes and the image itself. Download and unzip image folder before its use.
+  - Visualizer: to visualize the images with its correspondent classes and segmentation masks. Download and unzip image folder, then clone the ODIN visualizer tool repository (https://github.com/rnt-pmi/odin) in this folder. 
```

Comment:
why change from install to clone? it can be installed...

Reply:
Because I tried to download and install the Odin repository and I tried to install it without success. The installation process resulted in errors with pycocotools, which could not be installed. Instead, by simply cloning the repository (and changing a few lines, that we aim to update), the notebooks still work correctly. 

As a matter of fact, because of all the installation issues that we know occur when attempting the Odin installation, within the research group we resorted to creating a proper docker environment where only Odin is installed, in order not to alter all the library versions on the dockers we usually utilize, thus not creating compatibility issues with all the other services we use.

Reply:
Ok